utils/src/ooinstall/cli_installer.py

```diff
@@ -164,8 +165,8 @@ def collect_hosts(oo_cfg, existing_env=False, masters_set=False, print_summary=T
             more_hosts = click.confirm('Do you want to add additional hosts?')
 
     if num_masters == 1:
-        master = next((host for host in hosts if host.master), None)
-        master.storage = True
+        master = next((host for host in hosts if host.is_master()), None)
+        master['roles'].append('storage')
```

Comment:
Might want to have the ability to split out storage and lb hosts separate from the master, especially since they are not currently containerized.